lib/DataIntegrityProof.js

```diff
@@ -164,19 +166,23 @@ export class DataIntegrityProof extends LinkedDataProof {
 
     // set default `now` date if not given in `proof` or `options`
     let date = this.date;
-    if(proof.created === undefined && date === undefined) {
-      date = new Date();
-    }
+    if(date === null) {
+    } else {
```

Comment:
It looks like there could be a case of `this.proof.created` being set.  I assume this is subclass behavior since `this.proof` isn't set here.  In that case, should `date === null` clear `proof.created`?

Reply:
Good catches, thanks. Addressed.